app/index.html

```diff
@@ -27,17 +27,32 @@
       <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
 
-    
-    <div class="hero-unit">
-      <h1>'Allo, 'Allo!</h1>
-      <p>You now have</p>
-      <ul>
-        <li><a href="https://html5boilerplate.com/">HTML5 Boilerplate</a></li>
-        <li><a href="http://libsass.org/">Sass</a></li>
-        <li><a href="http://modernizr.com/">Modernizr</a></li>
-      </ul>
+    <div class="Nav-bar">
+      <div class="logo">MarikeAnne</div>
+      <div class="nav">
+        <a href="/about" class="">About</a>
+        <a href="/portfolio" class="">Portfolio</a>
+        <a href="/resume" class="">Resume</a>
+        <a href="/contact" class="">Contact</a>
+      </div>
     </div>
+
     
+    <div class="main-content">
+      <h1>Hello.</h1>
+      <h2>I am Marike, a graphic designer turned UI/UX designer.</h2>
+      <p>Over the years I have worked with various clients creating everything from wine labels, chocolate packaging to mobile apps and websites. I am passionate about my job. I love creating beautiful and impactful designs.</p>
```

Comment:
Dit help (vir github/pull requests) as jy die lyn kort hou (met enters in die kode)